src/Microsoft.TestPlatform.ObjectModel/Utilities/XmlRunSettingsUtilities.cs

```diff
@@ -87,6 +87,70 @@ public static bool ContainsDataCollector(IXPathNavigable runSettingDocument, str
         }
 
         /// <summary>
+        /// Check data collector exist with friendly name
+        /// </summary>
+        /// <param name="runSettingDocument"> XPathNavigable representation of a runsettings file </param>
+        /// <param name="dataCollectorFriendlyName"> The data Collector friendly name. </param>
+        /// <returns> True if there is a datacollector configured. </returns>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1054:UriParametersShouldNotBeStrings", MessageId = "1#")]
+        public static bool ContainsDataCollectorWithFriendlyName(IXPathNavigable runSettingDocument, string dataCollectorFriendlyName)
```

Comment:
`ContainsDataCollectorWithFriendlyName` can be moved in as a private static method in EnableCodeCoverageArgumentProcessor.

Reply:
Fixed in next commit.